bears/rest/RstcheckBear.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class RstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file,
+            code_block_language_ignore: str,):
+        """
+        A Bear for rstcheck. (https://pypi.python.org/pypi/rstcheck)
+        Rstcheck checks syntax of reStructuredText and code blocks
+        nested within it.
+        Supports bash, c, cpp, json, python and rst.
+
+        :param code_block_language_ignore: comma seperated list for languages
+                                           of which code blocks should be
+                                           ignored.(bash,c,cpp,json,python,rst)
+        """
+
+        for error in check(''.join(file), filename=filename,
+                           ignore=code_block_language_ignore.split(',')):
```

Comment:
Why ask for a string and do a split() when you can directly ask for a list and let coala handle the parsing ?